graphql/chat/mutations.ts

```diff
@@ -27,8 +27,8 @@ export class MutateChatResolver {
             },
         };
 
-        await getOrCreateConversation(matchees, conversationInfos);
-        return true;
+        const conversation = await getOrCreateConversation(matchees, conversationInfos);
+        return conversation.id;
```

Comment:
LGTM, lets just check this with @Jonasdoubleyou, hence the "convention" in this code base is to return true. But in this case seems valid to return the conversation.id as we need it in the frontend to access the created convo.

Reply:
Yeah that convention is mainly for "void" methods